src/OSPSuite.UI/Controls/UxButtonRepositories.cs

```diff
@@ -1,32 +1,27 @@
-﻿using DevExpress.XtraEditors.Controls;
-using OSPSuite.Assets;
+﻿using OSPSuite.Assets;
 using OSPSuite.UI.RepositoryItems;
 
 namespace OSPSuite.UI.Controls
 {
-   public class UxAddButtonRepository : UxRepositoryItemButtonEdit
+   public class UxAddButtonRepository : UxRepositoryItemButtonImage
```

Comment:
I have also defined those repo to use images instead of + and - etc.. looks a bit more modern I believe